src/Integration/Persistence/SolutionBindingSerializer.cs

```diff
@@ -64,7 +64,7 @@ public BoundSonarQubeProject Read(string configFilePath)
             return bound;
         }
 
-        public bool Write(string configFilePath, BoundSonarQubeProject binding, Predicate<string> onSuccessfulFileWrite)
```

Comment:
In future, please call in a comment why you are making non-obvious changes e.g. why this method is now returning an `Action` rather than a `Predicate`.

After wading through the changes, my guess is that it's because the only implementation always returned `true`, but it's just a guess.

Reply:
You're right, my bad. I'll reference it in the future.